src/scripts/main.js

```diff
@@ -1 +1,68 @@
 'use strict';
+
+const body = document.querySelector('body');
+const message = document.createElement('div');
+
+function handleSuccess(result) {
+  message.setAttribute('data-qa', 'notification');
+  message.className = 'success';
+  message.innerText = result;
+  body.append(message);
+}
+
+function handleError(error) {
+  message.setAttribute('data-qa', 'notification');
+  message.className = 'warning';
+  message.innerText = error;
+  body.append(message);
+}
```

Comment:
you can create one function for message adding with `type` param and reuse it. 
please move `message` div creation to this function because it rewrites the message every time

Reply:
unction handler(type) {
  const message = document.createElement('div');

  message.setAttribute('data-qa', 'notification');

  if (type === 'result') {
    message.className = 'success';
    message.innerText = result;
  } else if (type === 'error') {
    message.className = 'warning';
    message.innerText = error;
  }

  body.append(message);
}

Reply:
а как это сделать у меня ошибки выводит тогда